### ISSUE TITLE
When will be the next release of @nativescript/firebase?

### ISSUE THREAD

**Issue by cupidchan** [EddyVerbruggen/nativescript-plugin-firebase#1834]
Firebase has been graded for a few rounds. Is there any plan to release the next version of this @nativescript/firebase plugin?

**Comment by jkod** [comment]
From https://twitter.com/NativeScript: "We have rewritten it here (https://github.com/NativeScript/firebase) with latest native dependencies - private at moment but will become public soon and we'll begin publishing some updates with it soon as well."

**Comment by cupidchan** [comment]
> From https://twitter.com/NativeScript: "We have rewritten it here (https://github.com/NativeScript/firebase) with latest native dependencies - private at moment but will become public soon and we'll begin publishing some updates with it soon as well."

@jkod can you send the link of the twitter you mentioned? I cannot find that in the NativeScript twitter home page... And the https://github.com/NativeScript/firebase gives me a 404 error. Is that the same as you get too?

**Comment by jkod** [comment]
> > From https://twitter.com/NativeScript: "We have rewritten it here (https://github.com/NativeScript/firebase) with latest native dependencies - private at moment but will become public soon and we'll begin publishing some updates with it soon as well."
> 
> @jkod can you send the link of the twitter you mentioned? I cannot find that in the NativeScript twitter home page... And the https://github.com/NativeScript/firebase gives me a 404 error. Is that the same as you get too?

@cupidchan Yes, it was reply to a question on the roadmap update - here it is: https://twitter.com/NativeScript/status/1430201839341883394?s=20 

As it is private we all get a 404 error but hopefully that will change soon :-)

**Comment by 7ammer** [comment]
Seems to be active now...
https://github.com/NativeScript/firebase
